source/develop/release-management/features/virt/XenToOvirt.html.md

```diff
@@ -34,3 +34,11 @@ We are logging to the Xen host after we copy the ssh key in order to check that
 - Select **'XEN (via EL)'** in the source select box
 - Select VDSM host from the 'Proxy Host' select box
 - Enter valid URI such as: `xen+ssh://user@xenhost`
+
+## Import VMs with Block disks
+Currently it is not possible to import Xen VMs with block disks via the oVirt site.
+As a workaround please follow these steps:
+- Make sure you have export domain active
+- Run the command: `virt-v2v-copy-to-local -ic xen+ssh://root@xenserver.com vmname`
```

Comment:
should we maybe write it as 
`virt-v2v-copy-to-local -ic xen+ssh://<root>@<xenserver.com> <vmname>`
to make it clear that the user needs to replace it with the specific properties in this environment?